### ISSUE TITLE
Emqx 3 crashed scheduled . 

### ISSUE THREAD

**Issue by yulindong** [emqx/emqx#3574]
**Environment**:

- emqx 3.0 opensource
- Cent OS 7
- Kernel : 3.10.0 x86_64

Our emqx broker use default config, no cluster. It's been running stably for a year.
From June 2020, An crashed error is logged every 2 hours. The log in erlang.log :

```
2020-07-04 22:07:46 =CRASH REPORT====
  crasher:
    initial call: gen_rpc_acceptor:init/1
    pid: <0.21073.0>
    registered_name: 'gen_rpc.acceptor.127.0.0.1:35520'
    exception exit: {timed_out_waiting_for_auth,[{gen_statem,loop_event_result,9,[{file,"gen_statem.erl"},{line,1158}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
    ancestors: [gen_rpc_acceptor_sup,gen_rpc_sup,<0.1358.0>]
    message_queue_len: 1
    messages: [{tcp_closed,#Port<0.2820>}]
    links: [<0.1361.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 27
    reductions: 25837
  neighbours:
2020-07-04 22:07:46 =SUPERVISOR REPORT====
     Supervisor: {local,gen_rpc_acceptor_sup}
     Context:    child_terminated
     Reason:     timed_out_waiting_for_auth
     Offender:   [{pid,<0.21073.0>},{id,gen_rpc_acceptor},{mfargs,{gen_rpc_acceptor,start_link,undefined}},{restart_type,temporary},{shutdown,5000},{child_type,worker}]
```

And every five or six days, the broker is crashed. We could start broker again, with scheduled error as described above.

I know emqx has alive check every 15 minutes. The crash occur at alive check time point.

I wonder why this is happening.What should I do. 

Thanks for help

**Comment by terry-xiaoyu** [comment]
This might be a issue that has been fixed in emqx 4.0. See https://github.com/emqx/gen_rpc/pull/5

You could upgrade to 4.0.7 if possible, or you can build your emqx 3.0 by cloning the source code and applying the code change I mentioned above.

**Comment by yulindong** [comment]
Thanks. I will take a upgrade
